net/wg-installer/wg-server/lib/wg_functions.sh

```diff
@@ -40,6 +51,14 @@ wg_register () {
 	local mtu=$2
 	local public_key=$3
 
+	if [ "$(wg_key_exists $public_key)" == "1" ]; then
```

Comment:
== is a bashism IIRC.

Reply:
Thanks. What do you think about the newest change?